LICENSE

```diff
@@ -1,3 +1,10 @@
+The source code is licenced under Appache Licence Version 2.0 (see below).
```

Comment:
If I hadn't known about the addition of the nui editor icons I would have assumed that everything is under the Apache license. The credits file is not that obvious to find, as it is in a sub folder.

Maybe we should create a file called ASSET_LICENSES that contains the reference to the credits file, pretty much like the section I added to the LICENSE file.